sdk/eidolon_ai_sdk/apu/conversational_apu.py

```diff
@@ -133,6 +139,12 @@ async def schedule_request(
             if self.record_memory:
                 await self.memory_unit.storeMessages(call_context, conversation_messages)
             conversation.extend(conversation_messages)
+
+            # EDIT: get relevant mem0 memories and extend the conversation history with them
+            if self.longterm_memory_unit:
+                longterm_memories = self.longterm_memory_unit.searchMemories(call_context, conversation_messages)
+                conversation.extend(longterm_memories)
```

Comment:
Does this work well just throwing the messages at the end?

I would have assumed we want to update the system message with the memories. I suppose this is something we can tweak with better evals and shouldn't block our initial impl merge.

Reply:
Not sure if this should be an optional feature - should the user be able to specify whether they want to include the mem0 content as part of the system message? I agree that this customizability probably doesn't need to be part of the initial version